x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/expandable_section/expandable_section.tsx

```diff
@@ -82,54 +82,58 @@ export const ExpandableSection: FC<ExpandableSectionProps> = ({
       hasBorder
     >
       <div className="mlExpandableSection">
-        <EuiFlexGroup justifyContent="spaceBetween" gutterSize="none">
-          <EuiFlexItem grow={false}>
-            <EuiButtonEmpty
-              onClick={toggleExpanded}
-              iconType={isExpanded ? 'arrowDown' : 'arrowRight'}
-              iconSide="right"
-              flush="left"
-              data-test-subj={`mlDFExpandableSection-${dataTestId}-toggle-button`}
-            >
-              {title}
-            </EuiButtonEmpty>
+        <EuiFlexGroup justifyContent="spaceBetween" gutterSize="none" alignItems="center">
+          <EuiFlexItem>
+            <EuiFlexGroup gutterSize="l">
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmpty
+                  onClick={toggleExpanded}
+                  iconType={isExpanded ? 'arrowDown' : 'arrowRight'}
+                  iconSide="left"
+                  flush="left"
+                  data-test-subj={`mlDFExpandableSection-${dataTestId}-toggle-button`}
+                >
+                  <EuiText size="s" color="default">
+                    <h5>{title}</h5>
```

Comment:
Is there anything we can do with the styling of the tab title? I feel it gets sort of lost as the font size / styling is very similar to the rest of the tab header content. And does it need to be `h5`? I suspect that breaks accessibility rules about levels of `h` elements if the page doesn't have `h1` to `h4` already.

<img width="503" alt="image" src="https://user-images.githubusercontent.com/7405507/221222689-af223ae4-d19f-46f1-ab9d-2d74e454c614.png">

Reply:
Latest styling edits look good here!